tools/clang/test/CodeGenSPIRV/lit.local.cfg

```diff
@@ -1 +1,2 @@
-config.unsupported = True # HLSL Change - Disable lit suites.
+config.unsupported = True # HLSL Change - Disable and hide these lit suites.
+config.suffixes = []
```

Comment:
It may not matter, but I wonder why some disable and hide as here, while others such as CodeGenHLSL above just hide?

Reply:
The TAEF batch tests like CodeGenHLSL are running through the clang-taef suite, so they are not disabled, I am just hiding them from the LIT discovery scanner so they don't show up as unsupported tests. I don't think CodeGenSPIRV tests are included in our TAEF runs.